javascript/javascript1/week2/index.js

```diff
@@ -0,0 +1,68 @@
+// Flight booking fullname function
+
+const useFormalName = "Lord" + getFullname;
+
+getFullname("Benjamin", "Hughes", useFormalName);
+let fullname1  = "Benjamin";
+let fullname2  = "Hughes";
+function getFullname(fullname1, fullname2) {
+  if (useFormalName) {
+  return ("Lord" + " " + fullname1 + " " + fullname2)
+} else {
+  return (fullname1 + " " + fullname2)
+}
+}
+getFullname(fullname1, fullname2);
+/* the getFullname for women will work the similar way: we need to create boolean variable useWomenName and if it's true
+we should use Queen as a formal way instead of Lord.
```

Comment:
Yes, this is correct! Note though that it should be a boolean **parameter**, not a variable, since then you can change its value each time you call the function. So, this would be the function if we took into account the sex of the individual:
```javascript
function getFullname(firstname, surname, useFormalName, isMale) {
  ...
}
```
where `useFormalName` and `isMale` are boolean values.